Source/WebKit/UIProcess/WebURLSchemeTask.h

```diff
@@ -101,8 +100,7 @@ class WebURLSchemeTask : public API::ObjectImpl<API::Object::Type::URLSchemeTask
     Ref<WebURLSchemeHandler> m_urlSchemeHandler;
     RefPtr<WebProcessProxy> m_process;
     WebCore::ResourceLoaderIdentifier m_resourceLoaderID;
-    Markable<WebPageProxyIdentifier> m_pageProxyID;
-    Markable<WebCore::PageIdentifier> m_webPageID;
+    WeakPtr<WebPageProxy> m_page;
```

Comment:
We support calling WebURLSchemeTask functions from non-main threads.  I think we need to make this thread safe.  WKURLSchemeHandler should probably be made to work on loads from different processes as well.  So this change introduces a threading issue and the use of m_process isn't quite correct.

Reply:
I think the changes in this class are wrong for more than one reason. I undid this part.